rotkehlchen/tests/api/test_uniswap.py

```diff
@@ -1213,3 +1214,96 @@ def test_get_events_history_get_all_events_empty(
             result = assert_proper_response_with_result(response)
 
     assert result == {}
+
+
+@pytest.mark.parametrize('ethereum_accounts', [["0x3ba6eb0e4327b96ade6d4f3b578724208a590cef"]])
+@pytest.mark.parametrize('ethereum_modules', [['uniswap']])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_get_uniswap_trades_history_me(
+        rotkehlchen_api_server,
+        ethereum_accounts,  # pylint: disable=unused-argument
+        rotki_premium_credentials,  # pylint: disable=unused-argument
+        start_with_valid_premium,  # pylint: disable=unused-argument
```

Comment:
If you don't use them I think they don't need to be added to the arguments, as long as they are used by at least one argument/fixture. And they should all be used by `rotkehlchen_api_server`